atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/SingleBackendCommand.java

```diff
@@ -39,17 +38,16 @@ public Integer call() {
         }
```

Comment:
We're printing the stack trace when catching the exception. This makes baseline not happy. Do we want to keep doing that?